static/src/javascripts/bootstraps/enhanced/notifications.js

```diff
@@ -7,13 +7,12 @@ define([
     'common/utils/config',
     'common/utils/storage',
     'common/utils/ajax',
-    'common/utils/template',
     'common/utils/robust',
     'common/views/svgs',
     'common/modules/user-prefs',
-    'text!common/views/ui/notifications-follow-link.html',
-    'text!common/views/ui/notifications-explainer.html',
-    'text!common/views/ui/notifications-permission-denied-message.html',
+    'tpl!common/views/ui/notifications-follow-link.html',
```

Comment:
I'm just not certain about these kind of NGW-specific abbreviations. It seems to be adding proprietary knowledge to a codebase that really doesn't need it. It's not a point I'd live or die on, but this is probably not something I would choose myself.